osquery/tables/system/windows/drivers.cpp

```diff
@@ -241,16 +281,16 @@ QueryData genDrivers(QueryContext& context) {
     row.GetString("Manufacturer", r["manufacturer"]);
     row.GetString("DriverProviderName", r["provider"]);
 
-    bool isSigned;
-    row.GetBool("IsSigned", isSigned);
-    r["signed"] = isSigned ? INTEGER(1) : INTEGER(0);
+    bool is_signed;
+    row.GetBool("IsSigned", is_signed);
```

Comment:
Is there any possibility of `GetBool` to fail? Should we check the status?

Reply:
Def, I'll add in the status checks for the rest of the WMI queries. For this one in particular I'm just rolling the check into the ternary, do you think it's worth having a `VLOG(1)` if the fetch fails? I honestly don't think this is something that will happen often, if at all >.>

Reply:
Ah, nvm, I'm already checking and `VLOG(1)`'ing below for the inf name, it'll be consistent if I do it here also